src/modules/Resources/devices.types.ts

```diff
@@ -29,7 +29,7 @@ interface DeviceCreateInfoBasic {
   /**
    * Description of the device.
    */
-  description?: string | void;
+  description?: string | null;
```

Comment:
Not sure why `void` was here, but it returns `null` sometimes if the device was created without `description`